src/main/java/io/split/android/client/service/sseclient/PushNotificationManager.java

```diff
@@ -52,6 +63,13 @@ private void connectToSse(String token, List<String> channels) {
         mSseClient.connect(token, channels);
     }
 
+    private void scheduleSseDownNotification() {
```

Comment:
Can we rename this to `resetSSEKeepAliveTimer` or similar?

Reply:
Sure. Done!